demo/index.js

```diff
@@ -5,6 +5,11 @@ import App from './app'
 
 import './index.html'
 
+if (process.env.NODE_ENV !== 'production') {
+  const {whyDidYouUpdate} = require('why-did-you-update');
+  whyDidYouUpdate(React);
```

Comment:
do you mind removing this block?  This should only be used to profile performance as a means to an end.  I don't want to add a dependency that profiles performance (i.e. it should only be used when performing dev work)